### PR TITLE
Modified readmes to point to Track 2 content

### PR DESCRIPTION
**Pull request by lilyjma**
Updated readmes with a link to corresponding Track 2 content in following folders:

- https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/Microsoft.Azure.EventHubs/Rbac/ManagedIdentityWebApp
- https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK
- https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole
- https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/Microsoft.Azure.EventHubs/SampleEphReceiver
- https://github.com/Azure/azure-event-hubs/tree/master/samples/DotNet/Microsoft.Azure.EventHubs/SampleSender

We'll be doing this exercise for Java samples as well.

### CODE REVIEW

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (ramya-rao-a) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:3]: @spelluru 

The linked page does not talk about "how to run this sample". Maybe this was meant to be more like "how to assign roles"?

```suggestion
For information on Role based access (RBAC) in Event Hubs, and to assign roles, see [Authorize access to Event Hubs resources using Azure Active Directory](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-role-based-access-control).
```

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (ramya-rao-a) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:7]: Did we run this by @jsquire? Perhaps he has some common text around DefaultAzureCredential we can re-use? I'd much rather point to a common place that talks about DefaultAzureCredential than explain its inner workings here

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md
Comment (ramya-rao-a) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:40]: The note should be made available much earlier before leading the user down the path of Track 1 package

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md
Comment (ramya-rao-a) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:42]: Avoid using `this` or `here`  or `link` as link text. This hinders accessibility for screenreader users when they try to go over all links on the page. When doing so, link texts are read out loud. Hearing "this" a bunch of times gives no context as to what the link would be

**Review by ramya-rao-a — COMMENTED** [review]
Can we move the moving of files to a separate PR? Tackling readme updates separately than moving files would help with faster review

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (jsquire) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:7]: I'm not sure that detail around the specifics of `Azure.Identity` makes a ton of sense here, as it's not germane to the sample.   I may suggest going with something that just extols the benefits.  As a rough suggestion, maybe something, like:

_"Authorization was one of the most consistent areas of feedback for the Azure SDKs from our customers.  Libraries for different Azure services have not had a consistent approach to authorization, roles, many developers have felt that the learning curve was difficult, and the APIs did not offer a good, approachable, and consistent story for integrating with Azure Active Directory principals._

_The [`Azure.Identity`](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/README.md#azure-identity-client-library-for-net) library is intended to address that feedback and provide an approachable and flexible authentication experience across the Azure SDKs, including the `Azrue.Messaging.Eventhubs` library."_

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md
Comment (jsquire) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:51]: nit:  The branding is "Event Hubs namespace" which contains zero or more "Event Hub" instances.    Admittedly, I've seen quite a lot of inconsistent capitalization for `Event Hub` so you could probably get away with lowercase there if you were so inclined.

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md
Comment (jsquire) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:49]: I think you want to say ".NET Core SDK" here...

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (jsquire) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:5]: nit:  maybe instead of "new" we go with something like "current generation" or "current"?    New is only "new" for so long.... I would think we want to position this as "the current thing that we should use".

**Comment by jsquire** [comment]
For the `ControlAndDataPlane` ... series,  I think we want to normalize on `Management Plane` and `Data Plane` as the terminology.   Even the T1 libraries are "management" libraries and using that nomenclature.

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (lilyjma) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:3]: @spelluru gentle ping

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (lilyjma) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:7]: Yeah, my intent was to point out some benefits of how Track 2 does auth so people would actually click on the link :D I think what Jesse has makes sense here.

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md
Comment (lilyjma) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:51]: I'll use `Event Hub` since that's what the official microsoft docs use

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (lilyjma) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:7]: Tweaking Jesse's comment a bit: 

Authorization was one of the most consistent areas of feedback for the Azure SDKs from our customers. Libraries for different Azure services did not had a consistent approach to authorization, roles, etc. The APIs also did not offer a good, approachable, and consistent story for integrating with Azure Active Directory principals. As such, many developers felt that the learning curve was difficult.

The [Azure.Identity](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#azure-identity-client-library-for-net) library is intended to address that feedback and provide an approachable and flexible authentication experience across the Azure SDKs, including the `Azure.Messaging.EventHubs` library.

**Comment by lilyjma** [comment]
> For the `ControlAndDataPlane` ... series, I think we want to normalize on `Management Plane` and `Data Plane` as the terminology. Even the T1 libraries are "management" libraries and using that nomenclature.

will use `Management Plane` since that's more consistent to what I've been hearing. Also, there's a folder named `Management` in this repo.

**Comment by lilyjma** [comment]
@ramya-rao-a I've addressed the feedback in this PR. (Will submit another PR after this is merged for moving of files.)

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md
Comment (spelluru) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/AzureEventHubsSDK/README.md:3]: @lilyjma - sorry. what's the question?

File: samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md
Comment (ramya-rao-a) [inline samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:8]: What is the reason for us to point to "on behalf flow" here?

Comment (jsquire) [reply samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md:8]: I would question why we're pointing at the custom credential sample.  Unless I'm missing a nuance, the standard [ClientSecretCredential](https://docs.microsoft.com/dotnet/api/azure.identity.clientsecretcredential) would probably be the more relevant type; the custom role would likely be assigned to a normal service principal.

**Review by ramya-rao-a — APPROVED** [review]
Looks good to me. Please get a review from either @spelluru or @serkantkaraca before merging

**Review by spelluru — APPROVED** [review]
Looks good to me. 

One non-blocker. In DOCS, when we refer to service we use "E" and "H" as in "Event Hubs" or when we refer to a namespace "Event Hubs namespace" ('n' lower case but 'E' and 'H" uppercase). 

When referring to a hub instance or instances, we use "event hub" or "event hubs".

**Review by serkantkaraca — APPROVED** [review]

**Pull request merged**
